metadata.yaml

```diff
@@ -27,6 +33,14 @@ requires:
   prometheus-target:
     interface: http
   prometheus-rules:
-    interface: prometheus-rules
+    interface: prometheus_rules
   dashboards:
-    interface: grafana-dashboard
+    interface: grafana_dashboard
+  nrpe-external-master:
+    interface: nrpe_external_master
+  general-info:
+    interface: general_info
```

Comment:
The interface type in the NRPE charm is different: https://git.launchpad.net/charm-nrpe/tree/metadata.yaml#n24 How would this work?